core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -1238,8 +1219,7 @@ export class ConversationSkinComponent {
     ) {
       this.initLearnerAnswerInfoService(
         this.explorationId, this.explorationEngineService.getState(),
-        answer, interactionRulesService,
-        this.alwaysAskLearnerForAnswerDetails());
+        answer, interactionRulesService, false);
```

Comment:
Actually, I just realized why we had the "always ask" flag -- without it, you can't test this feature deterministically (e.g. in e2e tests), it ends up being probabilistic whether we ask.

Should we keep it instead? (Sorry, didn't realize it before.)

Reply:
Got it, I have reverted it.